qa/tasks/mgr/dashboard/test_mgr_module.py

```diff
@@ -52,6 +53,7 @@ def test_list_disabled_module(self):
         self.assertIsNotNone(module_info)
         self.assertFalse(module_info['enabled'])
 
+    @retry(on_exception=RuntimeError, tries=2, delay=0.5, logger=log)
```

Comment:
Why not using the logger from this file?
```suggestion
    @retry(on_exception=RuntimeError, tries=2, delay=0.5, logger=logger)
```

Reply:
thanks, fixed it